lib/resources/majors.tsx

```diff
@@ -2711,6 +2711,308 @@ const bsMolCell: Major = {
   ],
 };
 
+const bsNeuro: Major = {
+  degree_name: 'B.S. Neuroscience',
+  abbrev: 'B.S. Neuro',
+  department: 'AS Neuroscience',
+  total_degree_credit: 120,
```

Comment:
I dont know if we should leave this as 120 or the amount it says on the JHU website (70.5-72.5). This might be a good discussion to bring up in our next meeting.